index.bs

```diff
@@ -5679,39 +5674,34 @@ steps may throw an exception.
       <dt>If IsArray(|input|)</dt>
       <dd>
 
-        1. Let |len| be the ToLength(Get(|input|,
+        1. Let |len| be [=!=] ToLength(Get(|input|,
```

Comment:
Need ? or ! for the Get, not just the ToLength

Is there something guaranteeing that ToLength will succeed? What about this?

```js
{
  get length() {
    return { valueOf() { throw new Error("boo"); } };
  }
}
```